src/components/Frame/Frame.tsx

```diff
@@ -203,10 +210,14 @@ class Frame extends React.PureComponent<CombinedProps, State> {
       skipFocused && styles.focused,
     );
 
+    const skipTarget = skipToContentTargetId
+      ? skipToContentTargetId
+      : APP_FRAME_MAIN_ANCHOR_TARGET;
```

Comment:
if we drop the `skipToContentTargetId` prop this could become

```typescript
const skipTarget = skipToContentTarget
      ? skipToContentTarget.current.id
      : APP_FRAME_MAIN_ANCHOR_TARGET;
```

Reply:
Yes just saw that.